nautobot_firewall_models/choices.py

```diff
@@ -6,3 +6,9 @@
 IP_PROTOCOL_CHOICES = tuple((i, i) for i in PROTO_NAME_TO_NUM.keys())  # pylint: disable=consider-iterating-dictionary
 
 ACTION_CHOICES = (("allow", "allow"), ("deny", "deny"), ("drop", "drop"), ("remark", "remark"))
+
+# TODO
+# Thoughts:
+# - Basic case of 1-to-1 swapping out the source or the destination address for another
+# - Pooled case of 1-to-1 swapping where there is no fixed mapping (needed?
+MODE_CHOICES = (("one-to-one", "one-to-one"), ("one-to-many", "one-to-many"), ("remark", "remark"))
```

Comment:
This doesn't resonate as well to me, I think of thinks like static, dynamic, PAT, etc.. 

Perhaps take a quick look at rfc for a nat model https://datatracker.ietf.org/doc/rfc8512/

Reply:
I have to agree with Ken here. Also, I don't know if "mode" is necessarily the accurate word to use here. I think that type might make more sense and to have that be chosen first and that determine which fields are filled in, ie for a static NAT, I wouldn't need to specify ports.

Reply:
I have removed the mode field entirely, see my [comment](https://github.com/nautobot/nautobot-plugin-firewall-models/pull/92#issuecomment-1260784314) for an explanation.